bin/g5-transmitter.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+function usage
+{
+  echo "Usage:"
+  echo "cgm-transmitter [transmitter_id]"
+  echo
+  exit
+}
+
+function  generate_config
+{
+  echo "{\"transmitter_id\":\"${txId}\",\"sensor_code\":\"\",\"mode\":\"not-expired\",\"pump_units\":\"mg/dl\",\"fake_meter_id\":\"000000\"}" | jq . > $config
```

Comment:
if someone changes to this method and was using expired mode, then they will be switched to not-expired mode. They will have to edit the xdripjs.json file manually. Probably not a big deal, but might be worth mentioning right here with an echo to warn the user of this fact.

Reply:
Good point.  Right now the code should only generate a new config if the xdripjs.json doesn't exist at all so I was hoping it would only be defaulting in the case of never being set but could be something to call out for folks running in expired mode and then upgrading to the config file.